cmake/modules/Findkeyutils.cmake

```diff
@@ -13,7 +13,7 @@ INCLUDE(CheckCXXSymbolExists)
 
 # include dir
 
-find_path(KEYUTILS_INCLUDE_DIR keyutils.h NO_DEFAULT_PATH PATHS
+find_path(KEYUTILS_INCLUDE_DIR keyutils.h PATHS
```

Comment:
@bachp since by default, `find_path()` searches in `/usr/include`, could you remove it also?